infra/core/database/sqlserver/sqlserver-catalog.bicep

```diff
@@ -0,0 +1,129 @@
+param name string
```

Comment:
You don't need to create these files in core.  You can use the sqlserver.bicep file and pass the connectionStringKey to it.

Reply:
I resolved this in https://github.com/zedy-wj/eShopOnWeb/pull/2

Reply:
Thanks! We have merged #2 into this pr.